### PR TITLE
Creating a file for Mt. Vernon, OH

### LINKED ISSUE #20: Create fabgonzalez hometown file
## Mt. Vernon, OH
- [x] Create a branch
- [x] Create a file
- [x] Add a commit
- [ ] Create a Pull Request
- [ ] Have a conversation
- [ ] Get a :+1: 
  -[ ] :ship: it (a.k.a. merge)

### PR DESCRIPTION
**Pull request by fabgonzalez**
This PR resolves #20

### CODE REVIEW

File: fabgonzalez-mtvernon.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+## Mount Vernon, OH
+
+### Places to eat
+-Fiesta Mexicana
+### Fun things to do
```
Comment (adrianosimoes) [inline fabgonzalez-mtvernon.md:7]: @fabgonzalez intersting.

File: fabgonzalez-mtvernon.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+-Camping
```
Comment (hamid914) [inline fabgonzalez-mtvernon.md]: 👍

File: fabgonzalez-mtvernon.md (hunk 1 above)
Comment (ryaneva) [inline fabgonzalez-mtvernon.md:7]: @fabgonzalez test comment :-)

**Comment by fabgonzalez** [comment]
This looks :+1:

**Pull request merged**